### PR TITLE
TNO-2444: AV Overview adjustments

### PR DESCRIPTION
**Pull request by jdtoombs**
- update legend colours to match 
- update scripts slightly for better styling

### CODE REVIEW

File: libs/net/dal/Migrations/1.0.135/Up/PostUp/00-WeekdayTemplate.sql [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+UPDATE public.report_template SET
+"body" = '
+@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.ReportEngineAVOverviewModel>
+@using System
+@using System.Linq
+@using TNO.Entities
+<style>
+a { text-decoration:none; }
+.video-icon { fill: #702828; height: 20px; width: 20px; }
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.135/Up/PostUp/00-WeekdayTemplate.sql:12]: update styling

File: libs/net/dal/Migrations/1.0.135/Up/PostUp/00-WeekdayTemplate.sql [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        var time = item.ItemType switch
+        {
+          AVOverviewItemType.Ad => "",
+          _ => $"{item.Time}",
+        };
+        var summary = String.IsNullOrEmpty(item.Summary) ? "---" : item.Summary.Replace("\n", "<br/>");
+        <tr style="border-bottom: 1px solid #DED9DA;">
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@placement</td>
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px;">@(time?.Length == 8 ? time.Substring(0, 5) : time)</td>
+          <td style="border-bottom: 1px solid #DED9DA; font-size:.9rem; padding:2px 2px 2px 10px; display: flex;">
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.135/Up/PostUp/00-WeekdayTemplate.sql:71]: flex layout for better display / alignment

File: libs/net/dal/Migrations/1.0.135/Up/PostUp/01-WeekendTemplate.sql [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  }
+  td.head {
+    font-weight: bold;
+  }
+  .small {
+    text-align: center;
+    width: 50px;
+  }
+
+.video-icon { fill: #702828; height: 20px; width: 20px; }
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.135/Up/PostUp/01-WeekendTemplate.sql:37]: styling updates

File: libs/net/dal/Migrations/1.0.135/Up/PostUp/01-WeekendTemplate.sql [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        var time = item.ItemType switch
+        {
+          AVOverviewItemType.Ad => "",
+          _ => $"{item.Time}",
+        };
+        var summary = String.IsNullOrEmpty(item.Summary) ? "---" : item.Summary.Replace("\n", "<br/>");
+        <tr>
+          <td class="small">@placement</td>
+          <td class="small">@(time?.Length == 8 ? time.Substring(0, 5) : time)</td>
+          <td style="display: flex">
```
Comment (jdtoombs) [inline libs/net/dal/Migrations/1.0.135/Up/PostUp/01-WeekendTemplate.sql:83]: flex display

File: libs/net/dal/Migrations/1.0.135/Up/PostUp/00-WeekdayTemplate.sql (hunk 2 above)
Comment (Fosol) [reply libs/net/dal/Migrations/1.0.135/Up/PostUp/00-WeekdayTemplate.sql:71]: I don't think flex is support in email.  Isn't this report supposed to be a fixed width and have no wrapping?

Comment (jdtoombs) [reply libs/net/dal/Migrations/1.0.135/Up/PostUp/00-WeekdayTemplate.sql:71]: Oh right I have another ticket for that - I will update

**Pull request closed without merging**